src/backend/vulkan/src/physical_device.rs

```diff
@@ -186,6 +186,12 @@ impl PhysicalDeviceFeatures {
                         .runtime_descriptor_array(
                             features.contains(Features::UNSIZED_DESCRIPTOR_ARRAY),
                         )
+                        .shader_storage_buffer_array_non_uniform_indexing(
```

Comment:
let's group it nicer. The `UNSIZED_DESCRIPTOR_ARRAY` should be after the descriptor indexing sub-features